lib/transcoder.ex

```diff
@@ -121,15 +122,28 @@ defmodule Membrane.Transcoder do
     {[spec: spec], state}
   end
 
+  defp maybe_override_input_stream_format(
+         builder,
+         %Membrane.RemoteStream{content_format: Membrane.MPEGAudio} = stream_format
+       ) do
+    builder
+    |> child(:stream_format_changer, %Membrane.Transcoder.StreamFormatChanger{
+      stream_format: stream_format
+    })
+  end
+
   defp maybe_override_input_stream_format(builder, nil) do
     builder
   end
 
   defp maybe_override_input_stream_format(builder, stream_format) do
+    Membrane.Logger.warning("""
+    The only input stream format that can be assumed is
+    `%Membrane.RemoteStream{content_format: Membrane.MPEGAudio}`, while you wanted to assume:
+    #{inspect(stream_format)}
+    """)
+
     builder
-    |> child(:stream_format_changer, %Membrane.Transcoder.StreamFormatChanger{
-      stream_format: stream_format
-    })
```

Comment:
I suggest to raise here, as the code won't act as the user assumed

Reply:
done